dotnetv3/AutoScaling/Actions/AutoScalingWrapper.cs

```diff
@@ -0,0 +1,353 @@
+    /// </summary>
+    /// <param name="groupName">The name of the Auto Scaling group.</param>
+    /// <returns>A Boolean value indicating the success of the action.</returns>
+    public async Task<bool> EnableMetricsCollectionAsync(string groupName)
+    {
+        var listMetrics = new List<string>
+            {
+                "GroupMaxSize",
+            };
+
+        var collectionRequest = new EnableMetricsCollectionRequest
+        {
+            AutoScalingGroupName = groupName,
+            Metrics = listMetrics,
+            Granularity = "1Minute",
+        };
+
+        var response = await _amazonAutoScaling.EnableMetricsCollectionAsync(collectionRequest);
+        return response.HttpStatusCode == System.Net.HttpStatusCode.OK;
+    }
+
+    // snippet-end:[AutoScaling.dotnetv3.AutoScalingActions.EnableMetricsCollection]
+
+    // snippet-start:[AutoScaling.dotnetv3.AutoScalingActions.SetDesiredCapacity]
+    /// <summary>
+    /// Set the desired capacity of an Auto Scaling group.
+    /// </summary>
+    /// <param name="groupName">The name of the Auto Scaling group.</param>
+    /// <param name="desiredCapacity">The desired capacity for the Auto
+    /// Scaling group.</param>
+    /// <returns>A Boolean value indicating the success of the action.</returns>
+    public async Task<bool> SetDesiredCapacityAsync(
+        string groupName,
+        int desiredCapacity)
+    {
+        var capacityRequest = new SetDesiredCapacityRequest
+        {
+            AutoScalingGroupName = groupName,
+            DesiredCapacity = desiredCapacity,
+        };
+
+        var response = await _amazonAutoScaling.SetDesiredCapacityAsync(capacityRequest);
+        Console.WriteLine($"You have set the DesiredCapacity to {desiredCapacity}.");
+
+        return response.HttpStatusCode == System.Net.HttpStatusCode.OK;
+    }
+
+    // snippet-end:[AutoScaling.dotnetv3.AutoScalingActions.SetDesiredCapacity]
+
+    // snippet-start:[AutoScaling.dotnetv3.AutoScalingActions.TerminateInstanceInAutoScalingGroup]
+    /// <summary>
+    /// Terminate all instances in the Auto Scaling group in preparation for
+    /// deleting the group.
+    /// </summary>
+    /// <param name="instanceId">The instance Id of the instance to terminate.</param>
+    /// <returns>A Boolean value that indicates the success or failure of
+    /// the operation.</returns>
+    public async Task<bool> TerminateInstanceInAutoScalingGroupAsync(
+        string instanceId)
+    {
+        var request = new TerminateInstanceInAutoScalingGroupRequest
+        {
+            InstanceId = instanceId,
+            ShouldDecrementDesiredCapacity = false,
+        };
+
+        var response = await _amazonAutoScaling.TerminateInstanceInAutoScalingGroupAsync(request);
+
+        if (response.HttpStatusCode == System.Net.HttpStatusCode.OK)
+        {
+            Console.WriteLine($"You have terminated the instance {instanceId}");
```

Comment:
Do these console lines need punctuation?

Reply:
I don't think it needs a period at the end. However, I did add a colon. As in:

"You have terminated the instance: {instanceId}"